Makefile

```diff
@@ -117,12 +117,15 @@ go-mod: ## Cleanup and verify go modules
 		$(GO) mod tidy && \
 		$(GO) mod verify
 
+.PHONY: $(BUILD_DIR)/kustomize
```

Comment:
But why? The Makefile target should only trigger if `$(BUILD_DIR)/kustomize` does not exist in the filesystem.

Reply:
Hm, hm…I'm not able to reproduce the issue any more. I was not sure what caused the target to fail locally, I'll have to revisit once the issue comes up again.

/close